jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/bpmn2/BPMN2DataServiceImpl.java

```diff
@@ -147,8 +147,7 @@ public ProcessDefinition buildProcessDefinition(String deploymentId,String bpmn2
             BPMN2ProcessFactory.setBPMN2ProcessProvider(provider);
         }
         try {
-            ProcessDescRepoHelper threadLocalHelper = new ProcessDescRepoHelper();
-	        BPMN2DataServiceSemanticModule.setRepoHelper(threadLocalHelper);
+            ProcessDescRepoHelper threadLocalHelper = BPMN2DataServiceSemanticModule.getRepoHelper();
```

Comment:
so is this the main change to avoid the NPE? was there some sort of race condition that get in the way?